client/src/components/pages/LivePosts/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PostButton from 'components/utils/PostButton'
+import Post from 'components/utils/Post';
+import './style.css'
+
+class LivePosts extends Component {
+  state = {
+    livePosts: [],
+    error: '',
+  }
+
+  componentDidMount() {
+    //Mock Data for Testing the component//
+    const livePosts = [
+      { title: 'post 1', id: 1 },
+      { title: 'post 2', id: 2 },
+      { title: 'post 3', id: 3 },
+      { title: 'post 4', id: 4 },
+      { title: 'post 5', id: 5 },
+      { title: 'post 6', id: 6 },
+    ];
+    this.setState({ livePosts });
+  }
+
+  handleDelete = (id) => {
+    const { livePosts } = this.state;
+    //Testing Delete with Mock Data//
+    const deletedPost = livePosts.findIndex(post => post.id === Number(id));
+    this.setState((prevState) => {
+      prevState.livePosts.splice(deletedPost, 1);
+      return ({ livePosts: prevState.livePosts })
+    });
```

Comment:
@IsraaSulaiman Another way to write this is to filter out the deleted post.  Another thing you can do is to call setState() with the new state, instead of passing it a function that returns new state.  Just a little simpler, if you'd like:
```
const { livePosts } = this.state;
this.setState({
  livePosts: livePosts.filter( post => post.id !== Number(id)) //Filter will include every post that *does not* match the id
}
```